engine/runtime/src/main/scala/org/enso/compiler/codegen/IrToTruffle.scala

```diff
@@ -1755,7 +1755,7 @@ class IrToTruffle(
       location: Option[IdentifiedLocation],
       binding: Boolean = false
     ): CreateFunctionNode = {
-      val bodyBuilder = new BuildFunctionBody(arguments, body, None, binding)
+      val bodyBuilder = new BuildFunctionBody(arguments, body, None, false)
```

Comment:
I don't see any big downsides to doing that. And we can potentially get a substantial performance gain